src/main/java/net/mcreator/ui/validation/validators/UniqueNameValidator.java

```diff
@@ -126,7 +127,7 @@ private Predicate<String> textCheck(String name) {
 			return new ValidationResult(ValidationResultType.ERROR, L10N.t("validators.unique_name.empty", name));
 		if (otherNames.get().filter(textCheck(uniqueName)).count() > (isPresentOnList ? 1 : 0)
 				|| forbiddenNames.contains(uniqueName))
-			return new ValidationResult(ValidationResultType.ERROR, L10N.t("validators.unique_name.duplicate"));
+			return new ValidationResult(ValidationResultType.ERROR, L10N.t("validators.unique_name.duplicate", StringUtils.lowercaseFirstLetter(name)));

```

Comment:
No need to lowercase the argument given that the string starts with it.

Reply:
In fact it would be confusing to show differently what user entered in the field

Reply:
Sorry, it was a mistake, I wanted to uppercase the first letter because the name is the first word of the sentence, it should have the first letter uppercased.